pnpm-lock.yaml

```diff
@@ -3637,8 +3653,8 @@ packages:
   convert-source-map@2.0.0:
     resolution: {integrity: sha512-Kvp459HrV2FEJ1CAsi1Ku+MY3kasH19TFykTz2xWmMeq6bk2NU3XXvfJ+Q61m0xktWwt+1HSYf3JZsTms3aRJg==}
 
-  cookie@0.7.2:
-    resolution: {integrity: sha512-yki5XnKuf750l50uGTllt6kKILY4nQ1eNIQatoXEByZ5dWgnKqbnqmTrBE5B4N7lrMJKQ2ytWMiTO2o0v6Ew/w==}
+  cookie@0.7.1:
```

Comment:
How did you upgrade the deps? By changing the package.json and then running `pnpm i` or by running `pnpm update <package>`?

pnpm keeps messing with this cookie package and I cannot pin down why (cc @gzdunek, we've seen this before in the Electron dep update PR).

Reply:
By "messing with" I mean that in the recent Electron dep update PR, it updated it to 0.7.2 and now it gets downgraded to 0.7.1.

* https://github.com/gravitational/teleport/pull/51906#discussion_r1950845008
* https://github.com/gravitational/teleport/pull/51906/files#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR10613

Reply:
🙉 
Eh, this must be a bug in pnpm.

Reply:
I think you can keep it, it's tough editing the lockfile by hand. Alternatively, you can check if reverting lockfile changes, and then re-running `pnpm i` also results in those other packages being updated.

Reply:
So I recreated the lockfile by running `npm install`, but now it downgraded a random bunch of other packages...

Reply:
Like `foreground-child@3.3.1 -> foreground-child@3.3.0`, do we care about this one?

Reply:
Okay, regenerated the lockfile; cookie downgrade is gone, but these additional deps are still here. Not much more that I can do.